internal/kustomize/kustomize.go

```diff
@@ -80,3 +75,26 @@ func Render(
 	// Glue the manifests together
 	return manifests.CombineYAML(yamlManifests), nil
 }
+
+func BuildKustomizeOptions(
+	enableHelm bool,
+	loadRestrictor string,
+) *argoappv1.KustomizeOptions {
+	buildOptions := ""
+
+	if enableHelm {
+		buildOptions += "--enable-helm"
+	}
+
+	if loadRestrictor != "" {
```

Comment:
If `loadRestrictor` is empty - do we want to set it to `LoadRestrictionsRootOnly` so it's always set?

Reply:
Let's do that, if the default changes the behaviour stays consistent. The refactor helps with your other comments as well 😄